### PR TITLE
Release 5 update

### PR DESCRIPTION
**Pull request by up2cat**
1) Fix the usage of GwFileToFilePack API call
2) Updated default sysConfig switches

### CODE REVIEW

File: glasswall/content_management/config_elements/sysConfig.py [hunk 1]
```diff
@@ -8,16 +8,20 @@ class sysConfig(ConfigElement):
...
     """
 
     def __init__(self, attributes={}, **kwargs):
         self.name = self.__class__.__name__
         self.attributes = attributes
+
         self.switches_module = switches.sys
         self.default_switches = [
             self.switches_module.interchange_type(value="sisl"),
             self.switches_module.interchange_pretty(value="false"),
```
Comment (AngusWR) [inline glasswall/content_management/config_elements/sysConfig.py:21]:
```suggestion
            self.switches_module.interchange_pretty(value="true"),
```

File: glasswall/libraries/archive_manager/archive_manager.py [hunk 2]
```diff
@@ -600,24 +603,24 @@ def unpack_directory(self, input_directory: str, output_directory: str, recursiv
                 delete_origin=delete_origin
             )
 
-    def pack_directory(self, input_directory: str, output_directory: str, file_type: str, raise_unsupported: bool = True, delete_origin: bool = False):
+    def pack_directory(self, input_directory: str, output_directory: str, file_type: str, add_extension: Optional[bool] = True, raise_unsupported: Optional[bool] = True, delete_origin: Optional[bool] = False):
         """ Pack a directory. Supported archive formats are: "7z", "bz2", "gz", "rar", "tar", "zip".
```
Comment (AngusWR) [inline glasswall/libraries/archive_manager/archive_manager.py:607]:
```suggestion
        """ Pack a directory. Supported archive formats are: "7z", "bz2", "gz", "rar", "tar", "xz", "zip".
```

**Review by AngusWR — CHANGES_REQUESTED** [review]

File: glasswall/content_management/config_elements/sysConfig.py (hunk 1 above)
Comment (AngusWR) [reply glasswall/content_management/config_elements/sysConfig.py:21]: It looked like this was the intended change based on the change to line 11. If this is not the case and the default value for the `interchange_pretty` switch is still false then this suggested change can be ignored.

Comment (up2cat) [reply glasswall/content_management/config_elements/sysConfig.py:21]: It (line 11) was a comment as an example. Engine default value is `false`

**Comment by AngusWR** [comment]
change needed to `glasswall/__init__.py` line 8, `__version__` should be incremented

**Pull request merged**